src/chocolatey.resources/helpers/functions/Get-WebFile.ps1

```diff
@@ -159,6 +159,7 @@ param(
   $req.AllowAutoRedirect = $true
   $req.MaximumAutomaticRedirections = 20
   #$req.KeepAlive = $true
+  $req.AutomaticDecompression = [System.Net.DecompressionMethods]::GZip -bor [System.Net.DecompressionMethods]::Deflate

```

Comment:
And this is helpful - works in .net 2+ https://msdn.microsoft.com/en-us/library/system.net.decompressionmethods(v=vs.80).aspx